### PR TITLE
Connect to conference call after the message

### PR DESCRIPTION
**Pull request by sterlinghirsh**
If you stay on the phone call after the error message is read, you'll be asked to press a digit. If you do, you're connected to a conference call with everyone else.

# QA

Test https://github.com/iFixit/ifixit/pull/33279. (This is a dependency of https://github.com/iFixit/ifixit/pull/33279 and testing that will test this.)

Connects https://github.com/iFixit/ifixit/pull/33279

### CODE REVIEW

**Comment by djmetzle** [comment]
That's it? CR :gear:

**Comment by sterlinghirsh** [comment]
I know right? I'm working on a few more quality of life fixes that I'm going to put in here too.

**Comment by djmetzle** [comment]
CR :exclamation:

**Comment by addison-grant** [comment]
Are you opening a corresponding PR on iFixit/ifixit? Could we modify the fakeAlert script to take in multiple phone numbers?

**Comment by jarstelfox** [comment]
CR 🌵

**Comment by sterlinghirsh** [comment]
Changed it so you have to press a digit because it kept connecting to people's answering machines on the conference call. Slightly more work but much better result.

File: Alertinator.php [hunk 1]
```diff
@@ -252,16 +249,25 @@ protected function sms(string $number, string $message): void {
...
-      $twiml->dial()->conference(self::CONFERENCE_NAME);
+
+      // Connect the user to the conference call only if they press a
+      // digit on the phone. This seems more reliable than answering machine
+      // detection.
+      $gather = $twiml->gather([
+         'action' => 'https://handler.twilio.com/twiml/EHf60b496ecfab6bd73cc23f13e97c8f66',
+         'numDigits' => 1,
+      ]);
+      $gather->say('Press any digit to connect to conference');
```
Comment (djmetzle) [inline Alertinator.php:270]: Is this going to `say` the whole alert message non-stop? Like "alert wharrgarll threshold one press any digit to connect"? You've been exercising this @sterlinghirsh so i'll defer to you, but is there a way to put in, say, a one second pause before the conference connect? Just a thought.

**Comment by djmetzle** [comment]
CR :phone:

**Comment by jarstelfox** [comment]
CR 🌵

File: Alertinator.php (hunk 1 above)
Comment (sterlinghirsh) [reply Alertinator.php:270]: It does say the whole thing, but it does have slight pauses on punctuation. I wouldn't mind putting in a 1s delay though.

**Comment by sterlinghirsh** [comment]
I added the 1 second pause.

**Comment by djmetzle** [comment]
CR :pause_button:

**Comment by batbattur** [comment]
Tested https://github.com/iFixit/ifixit/pull/33279#issuecomment-645609934 and worked nicely. 

QA 👍

**Pull request merged**